ILD/compact/ILD_l5_v11/TrackerDiskModuleOut.xml

```diff
@@ -0,0 +1,27 @@
+<lccdd>
```

Comment:
This seems to be a copy(?) of `../ILD_common_v02/TrackerDiskModuleOut.xml`. Could that be used instead of a dedicated copy here?

Reply:
fixed